test/unit/require-for-access.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import RequireForAccess, {__get__} from '../../src';
+
+describe('RequireForAccess', function() {
+  beforeEach(() => {
+    //
+  });
+
+  describe('rendering', () => {
+    beforeEach(() => {
+      // this.preventScroll = __get__('preventScroll');
+      // this.preventScroll.on = stub();
+      // this.preventScroll.off = stub();
+    });
+
+    it('should render the provided children when authorizeAccess() is true', () => {
+      const wrapper = shallow(<RequireForAccess>
+        <div className="secret-stuff"/>
+        <div className="secret-stuff2"/>
+      </RequireForAccess>, {context: {authorizeAccess() {return true;}}});
+
+      // we render a wrapper component in order to allow folks to be able to
+      // pass siblings as children and still have them all render
+      assert(wrapper.find('.react-access-valid').length === 1);
+
+      // we should also expect to see all the content provided as children
+      assert(wrapper.find('.secret-stuff').length === 1);
+      assert(wrapper.find('.secret-stuff2').length === 1);
+    });
+
+    it('should not render the provided content when authorizeAccess() is false', () => {
+      const wrapper = shallow(<RequireForAccess>
+        <div className="secret-stuff"/>
+        <div className="secret-stuff2"/>
+      </RequireForAccess>, {context: {authorizeAccess() {return false;}}});
+
+      // we render a wrapper component in order to allow folks to be able to
+      // pass siblings as children and still have them all render
+      assert(wrapper.find('.react-access-invalid').length === 1);
+
+      // we should also expect to see all the content provided as children
+      assert(wrapper.find('.secret-stuff').length === 0);
+      assert(wrapper.find('.secret-stuff2').length === 0);
+    });
+
+    it('should render the invalidAccessComponent when provided', () => {
+      const invalid = <span>You don't have access</span>;
```

Comment:
Yea, the fix is probably to escape it, but doesn't seem worth it in this case. Might be good to do in the future though. Something like `\'` perhaps?